### ISSUE TITLE
MLSA for Analytics - license not accepted error

### ISSUE THREAD

**Issue by scarolan** [mengesb/tf_chef_stack#3]
When I attempted to `terraform apply` I got this error:

```
module.chef-server.aws_instance.chef-server (remote-exec): Connecting to remote host via SSH...
module.chef-server.aws_instance.chef-server (remote-exec):   Host: 54.201.197.61
module.chef-server.aws_instance.chef-server (remote-exec):   User: centos
module.chef-server.aws_instance.chef-server (remote-exec):   Password: false
module.chef-server.aws_instance.chef-server (remote-exec):   Private key: true
module.chef-server.aws_instance.chef-server (remote-exec):   SSH Agent: true
Error applying plan:

2 error(s) occurred:

* Error running command 'rm -rf .analytics ; mkdir -p .analytics
if [ "0" == "true" ]
then
  touch .analytics/.license.accepted
elif [ "0" == "false" ]
then
  continue
elif [ 0 -gt 0 ]
then
  touch .analytics/.license.accepted
else
  continue
fi
[ -f .analytics/.license.accepted ] && echo "Chef MLSA accepted" || echo "Chef MLSA NOT accepted"
[ -f .analytics/.license.accepted ] && exit 0                    || exit 1
': exit status 1. Output: Chef MLSA NOT accepted

* dial tcp 54.201.197.61:22: i/o timeout

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

**Comment by mengesb** [comment]
`dial tcp 54.201.197.61:22: i/o timeout` looks like a communcation problem with the VM that was spun up.

**Comment by mengesb** [comment]
You also need to explicitly accept the Chef MLSA licensing. This consolidated plan was written as a MVP sort of item before that was a thing. I'll update it to handle this.

You can see it in the sub-project that this uses: https://github.com/mengesb/tf_chef_analytics/blob/master/variables.tf#L105

**Comment by mengesb** [comment]
This should have been addressed with PR #4
